MatrixSDK/Crypto/CrossSigning/MXCrossSigning.m

```diff
@@ -496,6 +500,39 @@ - (NSString *)makeSigningKey:(OLMPkSigning * _Nullable *)signing privateKey:(NSD
     return pubKey;
 }
 
+- (void)registerUserDevicesUpdateNotification
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(userDevicesDidUpdate:) name:MXDeviceListDidUpdateUsersDevicesNotification object:nil];
```

Comment:
We need to listen to the right object. See my remark about `postNotificationName:MXDeviceListDidUpdateUsersDevicesNotification` below.